R/pkg/NAMESPACE

```diff
@@ -281,6 +281,8 @@ exportMethods("%<=>%",
               "initcap",
               "input_file_name",
               "instr",
+              "isInf",
+              "isinf",
```

Comment:
I really appreciate the attempt to include R, though a question, why do we have `isInf` and `isinf`?

Reply:
Yeah, I think we should not have both `isInf` and `isinf`. Since we have `isNaN`, `isNull`..., maybe `isInf` is more consistent.

Reply:
the functions are case insensitive so i don't think we need both?